dbt/models/reporting/reporting.ratio_stats.py

```diff
@@ -1,237 +1,234 @@
-# type: ignore
```

Comment:
The diff in this file might look pretty gnarly, but it's almost all whitespace changes due to the fact that the code needed to be indented one more layer to nest inside the context of the `model()` function (more on that soon). If you hide whitespace in the diff, it should be easier to review.